ceph-iscsi.spec

```diff
@@ -75,7 +75,7 @@ Requires:       python3-configshell >= 1.1.fb25
 %endif
 %endif
 
-%if 0%{?rhel} == 7
+%if 0%{?rhel} <= 8
```

Comment:
```suggestion
%if 0%{?rhel}
```

Reply:
If `rhel` is undefined (non-RHEL system) it will evaluate to 0 and pass the `<=` check.

Reply:
Re-pushed with the fix.